ynr/apps/ynr_refactoring/tasks.py

```diff
@@ -1,22 +1,30 @@
 from django.conf import settings
-from django.core.management import CommandError
 from django.core.management import call_command as core_call_command
 from django_q.models import Schedule
 from django_q_registry import register_task
 
 
 def call_command(*args, **kwargs):
     if settings.ENABLE_SCHEDULED_JOBS:
-        try:
-            core_call_command(*args, **kwargs)
-        except SystemExit as e:
-            raise CommandError("Management Command Exited unexpectedly") from e
+        core_call_command(*args, **kwargs)
```

Comment:
This turned out not to be a great idea
https://github.com/django-q2/django-q2/blob/master/django_q/exceptions.py
We could special-case `TimeoutException` here, but this was mostly a guard against a theoretical problem that might happen. I think I'm inclined to just leave it like this tbh.